templates/lightning/index.html

```diff
@@ -28,7 +28,7 @@ <h2>DIAGNOSTIC</h2>
 <h2>BTC</h2>
 <div class='some-page-wrapper'>
     <div class='row'>
-        <a href="new_address">
+        <a href="/ln_wallet/new_address/bech32">
```

Comment:
the link should just be `/ln_wallet/new_address` in line with the previous comment

Reply:
with the change above theres an error. look at comment from above.